assets/js/pages/Users/generatePassword.js

```diff
@@ -0,0 +1,15 @@
+const DIGITS = '0123456789';
+const UPPERCASE_LETTERS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+const LOWERCASE_LETTERS = 'abcdefghijklmnopqrstuvwxyz';
+const SPECIAL_CHARACTERS = '~!@-#$%^&*()_+={}[]|:;<>,.?/';
+
+const PASSWORD_CHARACTERS =
+  DIGITS + UPPERCASE_LETTERS + LOWERCASE_LETTERS + SPECIAL_CHARACTERS;
+
+export const generatePassword = (
+  length = 16,
+  characters = PASSWORD_CHARACTERS
+) =>
+  Array.from(crypto.getRandomValues(new Uint32Array(length)))
```

Comment:
Does this guarantee that we will always get a password that  meets our requirements?

Reply:
No good point @rtorrero, there is a minimal chance that we have char repetition. 
I would tackle this in a follow-up pr